pkg/pillar/cmd/client/client.go

```diff
@@ -154,7 +154,6 @@ func Run() { //nolint:gocyclo
 
 	clientCtx := clientContext{
 		deviceNetworkStatus: &types.DeviceNetworkStatus{},
-		globalConfig:        &types.GlobalConfigDefaults,
```

Comment:
Should we still not include the spec map. It seems like its useful here

Reply:
Why is the spec map useful here??